.github/workflows/release.yml

```diff
@@ -1,4 +1,4 @@
-name: dxCompiler Release (Staging and Prod)
+name: Release dxCompiler
```

Comment:
I think the long name is why it isn't displaying correctly in the left panel here? 
https://github.com/dnanexus/dxCompiler/actions